spikeextractors/extractors/neuroscopeextractors/neuroscopeextractors.py

```diff
@@ -232,13 +235,10 @@ def __init__(self, resfile_path: PathType = None, clufile_path: PathType = None,
             n_clu = clu[0]
             clu = np.delete(clu, 0)
             unique_ids = np.unique(clu)
-
-            if not np.sort(unique_ids) == np.arange(n_clu + 1):  # some missing IDs somewhere
-                if 0 not in unique_ids:  # missing unsorted IDs
-                    n_clu += 1
-                if 1 not in unique_ids:  # missing mua IDs
-                    n_clu += 1
-                # If it is any other ID, then it would be very strange if it were missing...
```

Comment:
After a closer look at functional testing on the actual data, the outer `if` statement triggers 100% of the time, and also generally isn't needed in addition to the inner `if` statements.